cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -317,6 +319,9 @@ var createModeSubcommands = map[string]modeSubcommand{
 		ModeName: v1alpha1.SlurmMode,
 		Setup: func(clientGetter util.ClientGetter, subcmd *cobra.Command, o *CreateOptions) {
 			subcmd.Use += " [--ignore-unknown-flags]" +
+				" [--init-image IMAGE]" +
+				" [--init-image IMAGE]" +
```

Comment:
```suggestion
```

It seems it's duplicated.

Reply:
Ah, yes, thanks! It looks like after rebase missed :)

Reply:
Done

Reply:
Thanks!